framework/src/modules/interoperability/base_cross_chain_update_command.ts

```diff
@@ -196,8 +307,27 @@ export abstract class BaseCrossChainUpdateCommand<
 			);
 			return;
 		}
-		const command = commands.find(com => com.name === ccm.crossChainCommand);
-		if (!command) {
+
+		const crossChainCommand = crossChainCommands.find(com => com.name === ccm.crossChainCommand);
```

Comment:
Wouldn't it be cleaner to merge this branch with the unsupported module one? It would avoid quite a bit of code repetition

Reply:
sorry by branch I meant code branch, i.e. the check at line 283